github/orgs_teams.go

```diff
@@ -196,6 +196,9 @@ func (s *OrganizationsService) ListTeamMembers(ctx context.Context, team int, op
 // IsTeamMember checks if a user is a member of the specified team.
 //
 // GitHub API docs: https://developer.github.com/v3/orgs/teams/#get-team-member
+//
+// Deprecated: This API has been marked as deprecated in the Github API docs,
```

Comment:
Minor nitpick (for future reference), it's spelled "GitHub", not "Github". For consistency with the rest of the code/comments here.

Somewhat relevant: https://dmitri.shuralyov.com/idiomatic-go#for-brands-or-words-with-more-than-1-capital-letter-lowercase-all-letters.

Reply:
Whups, working with @willnorris, I usually catch that... sorry I missed it!